vocab_list/models.py

```diff
@@ -50,13 +50,20 @@ class VocabularyList(models.Model):
     )
 
     def load_tab_delimited(self, fd):
-        def clean(headword, gloss):
+        def clean(headword, gloss, *args):
```

Comment:
I guess you hit an issue where because your spreadsheet had more than two columns.  Is that so?

Reply:
That's right. We just wanted to deal with this in a way that didn't cause the process to fail.